src/exec/run-batch-submitter.ts

```diff
@@ -81,6 +81,10 @@ const requiredEnvVars: RequiredEnvVars = {
 const env = process.env
 const FRAUD_SUBMISSION_ADDRESS = env.FRAUD_SUBMISSION_ADDRESS || 'no fraud'
 const DISABLE_QUEUE_BATCH_APPEND = !!env.DISABLE_QUEUE_BATCH_APPEND
+const MIN_GAS_PRICE = parseInt(env.MIN_GAS_PRICE, 10) || 1
```

Comment:
Could consider making the default `0` so that you can submit to a local testnet without having eth (gasprice=0 is nice because anyone can submit txs without even an eth balance)